cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -109,13 +110,13 @@ private static String findNetwork(Compute compute, String project) throws IOExce
       NetworkList networkList = compute.networks().list(project).execute();
       networks = networkList.getItems();
     } catch (Exception e) {
-      handleRetryableExceptions(e);
-      throw e;
+      handleRetryableExceptions(e, ErrorTag.CONFIGURATION);
+      throw new DataprocRuntimeException(e, ErrorTag.CONFIGURATION);
```

Comment:
How do you know it's `CONFIGURATION`?

Reply:
The transient errors are being handled by handleRetryableException. Whatever the remaining exceptions are, I saw them under a larger umbrella of network configuration.

Reply:
Removed the CONFIGURATION tag. Depending on errors from this code path, we can probably tag it better in the future.

On a related note, shouldn't handlerRetryableException also handle IOException?